src/common/hotbar-store.ts

```diff
@@ -1,6 +1,7 @@
 import { action, comparer, observable, toJS } from "mobx";
 import { BaseStore } from "./base-store";
 import migrations from "../migrations/hotbar-store";
+import { v4 as uuid } from "uuid";
```

Comment:
nit: I prefer doing `import "uuid" from "uuid";` and then using it as `uuid.v4()` since that means that when looking at the uses I don't have to come back and checker which version we are importing.

Reply:
Changed (although other parts of codebase are using "v4 as uuid").

Reply:
thanks, I know. For my code I tend to use it.